classes/User.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+class User
+{
+    private $login;
+    private $password;
+
+    public function __construct($login, $password)
+    {
+        $this->login = $login;
+        $this->password = $password;
+    }
+
+    public static function createUser(string $login, string $password)
```

Comment:
do not overcomplicate. there is no logic in this static function. remove it and don't harden things